example/assets/subtitles.srt

```diff
@@ -0,0 +1,6 @@
+1
```

Comment:
What is this file?

Reply:
This file was a file used to generate subtitles when testing the subtitles filter - I forgot to remove from the example/assets. Will remove now. Thanks.